db/schema.rb

```diff
@@ -63,6 +63,16 @@
 
   add_index "date_ranges", ["event_id"], name: "index_date_ranges_on_event_id"
 
+  create_table "emails", force: :cascade do |t|
+    t.boolean  "hide_recipients"
+    t.string   "recipients"
+    t.string   "reply_to"
+    t.string   "subject"
+    t.string   "content"
+    t.datetime "created_at",      null: false
+    t.datetime "updated_at",      null: false
+  end
+
```

Comment:
Maybe it is just me, but why do we need a database table for emails? Considering the attributes I dont think its for drafts.